frontend/src/components/Election/Voting/VotePage.tsx

```diff
@@ -45,35 +47,26 @@ const VotePage = () => {
     let pages = election.races.map((race, i) => {
       let candidates = race.candidates.map(c=> ({...c, score: null}))
       return {
-        type: "ballot",
+        instructionsRead: false,
```

Comment:
I added a setting in #385 "require_instruction_confirmation", we should only show this when that is set. We also need to enable that in the settings page in this PR.

Reply:
Done!